R/formatter.r

```diff
@@ -1,3 +1,72 @@
+#' Number formatter: a generic formatter for numbers
+#'
+#' @return \code{number_format} returns a function with single parameter
```

Comment:
Can you please use markdown formatting, i.e. `` `number_format` ``

Reply:
corrected